Tools/eslint-config-cesium/index.js

```diff
@@ -20,6 +20,7 @@ module.exports = {
         'curly': 'error',
         'block-scoped-var': 'error',
         'eqeqeq': 'error',
+        "eol-last": ["error", "always"]
```

Comment:
Did you test this by running eslint?  It fails for me because of a missing comma here.

Once you fix it, there are a few errors you'll need to fix as well (because some of our generated files don't end in new lines)